scripts/website.js

```diff
@@ -76,6 +77,48 @@ const tests = [
   ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/schemas.test.js')).toString())
 ];
 
+function deleteAllHtmlFiles() {
+  try {
+    fs.unlinkSync('./index.html');
+  } catch (err) {
+    if (err.code !== 'ENOENT') {
+      throw err;
+    }
+  }
+  const foldersToClean = [
+    './docs',
+    './docs/tutorials',
+    './docs/typescript',
+    './docs/api',
+    './docs/source/_docs',
```

Comment:
those would likely need to be prefixed with `versionObj.versionedPath` (guessing by [this workflow](https://github.com/Automattic/mongoose/blob/ea855148c7ecdcfc60f46f286165617380083bc7/package.json#L99) and how this function is always called)

Reply:
I don't think these paths need to be prefixed with `versionedPath`. This `deleteAllHtmlFiles()` function is meant to replace `npm run docs:clean:stable`; which cleans out `index.html`, `docs/*.html`, etc. regardless of whether this is a versioned deploy or not.

In other words, the workflow for a versioned deploy is 1) clear all docs, 2) generate docs, 3) copy docs to `/tmp`, 4) clear `versionedPath`, 5) copy `/tmp` to `versionedPath`.  `deleteAllHtmlFiles()` takes care of step 1, not step 4.

Reply:
> just to be on a consistent page, this is the current workflow (to my understanding) Not entirely correct. 7.x and 6.x workflows don't check out gh-pages, so no steps 1/2. But otherwise correct.

as it said `if latest` - which applies to the master branch (currently 8.x) and `if versioned` it means the `X.x` branches (like `7.x` and `6.x`) - versioned uses gh-pages checkout at step 9